src/actions/userActions.js

```diff
@@ -17,14 +17,34 @@
     with this program; if not, write to the Free Software Foundation, Inc.,
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
-import { UPDATE_USER, REGISTERED, USER_PHONE_VERIFIED } from 'constants/userConstants';
+import {
+  SENDING_OTP,
+  OTP_SENT,
+  RESET_OTP_STATUS,
+  UPDATE_USER,
+  REGISTERED,
+  USER_PHONE_VERIFIED,
+  USER_EMAIL_VERIFIED,
+} from 'constants/userConstants';
 import { ADD_NOTIFICATION } from 'constants/notificationConstants';
 import { ACCOUNT_TYPES } from 'constants/accountsConstants';
 import { logEventAction } from 'actions/analyticsActions';
 import type { Dispatch, GetState } from 'reducers/rootReducer';
 import SDKWrapper from 'services/api';
 import { saveDbAction } from './dbActions';
 
+const sendingOneTimePasswordAction = () => ({
+  type: SENDING_OTP,
+});
+
+const otpPasswordSentAction = () => ({
```

Comment:
As other actions have relation to oneTimePassword and only this has "otp" short handle:
```suggestion
const oneTimePasswordSentAction = () => ({
```